compiler/rustc_codegen_llvm/src/gotoc/current_fn.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use crate::gotoc::GotocCtx;
+use crate::gotoc::Stmt;
+use rustc_hir::def_id::DefId;
+use rustc_middle::mir::BasicBlock;
+use rustc_middle::mir::Body;
+use rustc_middle::ty::Instance;
+use rustc_middle::ty::PolyFnSig;
+use rustc_middle::ty::TyCtxt;
+
+/// This structure represents useful data about the function we are currently compiling.
+pub struct CurrentFnCtx<'tcx> {
+    /// The GOTO block we are compiling into
+    pub block: Vec<Stmt>,
+    /// the current MIR basic block
+    current_bb: Option<BasicBlock>,
+    /// the codegen instance for the current function
+    instance: Instance<'tcx>,
+    /// the goto labels for all blocks
+    labels: Vec<String>,
+    /// the mir for the current instance
```

Comment:
Start with capital letters as in the other comments.

Reply:
Done